### DISCUSSION TITLE
how to set multiple roles to authN if a user with multiple roles ?

### DISCUSSION THREAD

**Discussion by 729902288** [PostgREST/postgrest#2932] [Q&A]
please, how to set **multiple roles to authN** if a user with multiple roles ?
example:
1、user：a user named john in users;
2、role：a department role  ra，a department role rb ，and grant ra,rb to authN(noinherit);
3、how to set role **ra,rb** etc(all) to authN ?

**Comment by wolfgangwalther** [comment]
Could you put your example into some SQL code and clearly demonstrate what you want to happen? I don't really understand what you are trying to achieve.
